### DISCUSSION TITLE
Error update Debian Err:8 https://packages.sury.org/

### DISCUSSION THREAD

**Discussion by bytecanarias** [oerdnj/deb.sury.org#2004] [General]
When I try to update the system I get the following error.

Err:8 https://packages.sury.org/apache2 bullseye InRelease
  Cannot initiate the connection to packages.sury.org:443 (2400:52e0:1e00::1082:1). - connect (101: Network is unreachable) Could not connect to packages.sury.org:443 (169.150.247.38), connection timed out
Err:9 https://packages.sury.org/php bullseye InRelease
  Cannot initiate the connection to packages.sury.org:443 (2400:52e0:1e00::1082:1). - connect (101: Network is unreachable)
Fetched 96.3 kB in 30s (3,176 B/s)
Reading package lists... Done
W: Failed to fetch https://packages.sury.org/apache2/dists/bullseye/InRelease  Cannot initiate the connection to packages.sury.org:443 (2400:52e0:1e00::1082:1). - connect (101: Network is unreachable) Could not connect to packages.sury.org:443 (169.150.247.38), connection timed out
W: Failed to fetch https://packages.sury.org/php/dists/bullseye/InRelease  Cannot initiate the connection to packages.sury.org:443 (2400:52e0:1e00::1082:1). - connect (101: Network is unreachable)
W: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by BMiZEL** [comment]
I see you closed the issue. How did you fix it @bytecanarias ?

I am having the same issue on Bookworm since a few weeks.

*Using a Static IP and I checked the DNS, which is working fine*

Ign:8 https://packages.sury.org/php bookworm InRelease
Err:8 https://packages.sury.org/php bookworm InRelease
  Cannot initiate the connection to packages.sury.org:443 (2400:52e0:1e01::879:1). - connect (101: Network is unreachable) Could not connect to packages.sury.org:443 (84.17.46.53), connection timed out
Fetched 57.3 kB in 38s (1523 B/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
W: Failed to fetch https://packages.sury.org/php/dists/bookworm/InRelease  Cannot initiate the connection to packages.sury.org:443 (2400:52e0:1e01::879:1). - connect (101: Network is unreachable) Could not connect to packages.sury.org:443 (84.17.46.53), connection timed out
W: Some index files failed to download. They have been ignored, or old ones used instead.

**Reply by bytecanarias** [reply]
My problem was generated by a block by Crowdsec, I don't know if you use that tool to block traffic to your computer. If not, check your firewall if you have one.

**Reply by BMiZEL** [reply]
that must be it, thank you so much. I did not honestly think to check there, as Crowdsec should block incoming and not outgoing connections...
